spring-web/src/main/java/org/springframework/web/SpringServletContainerInitializer.java

```diff
@@ -129,7 +129,7 @@
 	 * method will be invoked on each instance, delegating the {@code ServletContext} such
 	 * that each instance may register and configure servlets such as Spring's
 	 * {@code DispatcherServlet}, listeners such as Spring's {@code ContextLoaderListener},
-	 * or any other Servlet API componentry such as filters.
+	 * or any other Servlet API component such as filters.
```

Comment:
This is not a typo: "componentry" is semantically correct here.

Although the wording could be improved, the suggested change is grammatically incorrect.

Reply:
Thank you review but can I ask a question in this word "component" change to the "componentry" means, because componentry is not in the dictionary

Reply:
> This is not a typo: "componentry" is semantically correct here.
> 
> Although the wording could be improved, the suggested change is grammatically incorrect.

I am also not the English native speaker,I can not understand at the word add ry means what,so how to change the word can grammatically correct.

Reply:
what dictionary are you using?
https://www.dictionary.com/browse/componentry